config/initializers/active_record_belongs_to_required_by_default.rb

```diff
@@ -0,0 +1,4 @@
+# Be sure to restart your server when you modify this file.
+
+# Require `belongs_to` associations by default.
+Rails.application.config.active_record.belongs_to_required_by_default = true
```

Comment:
You could also just put a `null: false` condition on the foreign key, right? Or am I misunderstanding your usage here?